services/blockchain-indexer/shared/dataService/pos/validators.js

```diff
@@ -261,7 +261,8 @@ const updateValidatorListEveryBlock = () => {
 			if (block && block.transactions && Array.isArray(block.transactions)) {
 				block.transactions.forEach(tx => {
 					if (tx.module === MODULE.POS) {
-						if (tx.command === COMMAND.REGISTER_VALIDATOR) {
+						if (tx.command === COMMAND.REGISTER_VALIDATOR
+							|| tx.command === COMMAND.CHANGE_COMMISSION) {
```

Comment:
Alternative way of writing,

```suggestion
						if ([COMMAND.REGISTER_VALIDATOR, COMMAND.CHANGE_COMMISSION].includes(tx.command)) {
```